src/components/aktivitetskrav/vurdering/VurderAktivitetskravModal.tsx

```diff
@@ -29,18 +29,20 @@ export const VurderAktivitetskravModal = ({
 }: VurderAktivitetskravModalProps) => {
   return (
     <Modal
+      closeOnBackdropClick
       onClose={() => setModalOpen(false)}
       open={isOpen}
       aria-labelledby={texts.modalContentLabel}
+      header={{ heading: "" }}
```

Comment:
For å fortsatt ha lukk-knapp øverst til høyre har jeg lagt på tom heading (gjelder et par andre steder vi bruker Modal også). I disse casene ligger det også en header-tekst som en del av `Modal.Body`, men det krever en større refaktorering å flytte den ut hit. Kan se på å forbedre det i ny PR.

Reply:
Ja, høres ut som noe vi vil fikse på sikt 👍🏼

Reply:
https://github.com/navikt/syfomodiaperson/pull/1097